### PR TITLE
Adapt Azure TF example to use yamlencode

### PR DESCRIPTION
**Pull request by kke**
Makes the azure example from #53 utilize the `yamlencode` from #57 

Also adds a note about rebooting windows hosts until 1.1 is out.

### CODE REVIEW

**Comment by stevejr** [comment]
@kke - Can you add the file examples/terraform/README.md to the PR as it needs to be updated story state that Azure examples are available. Other than that it looks good

**Comment by kke** [comment]
> @kke - Can you add the file examples/terraform/README.md to the PR as it needs to be updated story state that Azure examples are available. Other than that it looks good

Added. Also noticed that OpenStack was missing and some of the examples still stated `yq` needs to be installed.

File: examples/terraform/README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 Working examples for using Terraform with Mirantis Launchpad. The scripts are provided as examples; you can change them to fit your specific configuration requirements or create your own.
 
 
-* [AWS](aws/README.md), a complete infrastructure example including VPC, LB, security groups, and other settings. 
-
+* [AWS](aws/README.md), a complete infrastructure example including VPC, LB, security groups, and other settings.
+* [Azure](azure/README.md), a complete exampe including VNET, LB, network security rules, and other settings.
```
Comment (stevejr) [inline examples/terraform/README.md]: Typeo:

```suggestion
* [Azure](azure/README.md), a complete example including VNET, LB, network security rules, and other settings.
```

**Comment by stevejr** [comment]
@kke - Do we need to highlight anywhere that the examples, at least the Azure Terraform one, does not setup any DTR related infra?

**Comment by kke** [comment]
> @kke - Do we need to highlight anywhere that the examples, at least the Azure Terraform one, does not setup any DTR related infra?

Seems like none of them do. We can add those in later, I think this one is good to go.

**Pull request merged**